src/app/vm/vm-sidebar/instance-group-selector/instance-group-selector.component.ts

```diff
@@ -28,7 +28,8 @@ export class InstanceGroupSelectorComponent {
   }
 
   public get groupName(): string {
-    return this.vm.instanceGroup && this.vm.instanceGroup.name;
+    const instanceGroup = this.vm && getInstanceGroup(this.vm);
+    return instanceGroup && instanceGroup.value;
```

Comment:
You use the same function in 2 places so and both of them are using `getInstanceGroup` why not to change the behavior of  `getInstanceGroup` to `getInstanceGroupName`?